samples/java-application-with-reflection/build.gradle

```diff
@@ -41,10 +41,11 @@
 
 plugins {
     id 'application'
-    id 'org.graalvm.buildtools.native'
+    id('org.graalvm.buildtools.native') version "${nativeGradlePluginVersion}"
```

Comment:
:x: instead of using a version here, the version should be declared in `settings.gradle` plugin management block, which would let you use dotted versions.